### ISSUE TITLE
Hosted gallery inaccessible with TalkBack

### ISSUE THREAD

**Issue by pepijndevos** [flutter/gallery#495]
**Describe the bug**
 
I was trying to evaluate Flutter for performance and accessibility, and when visiting the gallery on my phone, besides being disappointing in the performance,  I could not access the app at all. This seems like an obvious bug.

**To Reproduce**
1. On an Android phone go to settings->accessibility and enable TalkBack
2. Go to https://gallery.flutter.dev
3. Swipe right until the accessibility enable button is announced.
4. Double tap as instructed

**Expected behavior**

Accessibility should be enabled.

Instead it just processes the tap in some odd way, even if you double tap outside any active areas.

**Screenshots**

https://user-images.githubusercontent.com/168609/116807557-19892700-ab34-11eb-8a2f-2727ce1a6d0f.mp4

**Smartphone (please complete the following information):**
 - Device: Galaxy A51
 - OS: Android 11
 - Browser: Chome
 - Version: 90.0.4430.91

**Comment by MrBirb** [comment]
@yjbanov Does your PR address this one as well?

**Comment by guidezpl** [comment]
@yjbanov Can you take a look? I confirmed the mobile web version is completely inaccessible, even with the latest master https://gallery-staging-flutter-dev.firebaseapp.com/#/

**Comment by guidezpl** [comment]
This looks to be fixed in a recent staged release https://gallery-flutter-staging.web.app. As for performance, this is likely due to the HTML renderer being used on your mobile device, rather than the canvasKit renderer.
